dev/src/dev/debug_qp.clj

```diff
@@ -394,7 +391,7 @@
             (*printer* [::transformed-row @before row]))
           (rf result row)))))))
 
-(defn- default-debug-middleware
```

Comment:
The debug QP stuff in `dev.debug-qp` is currently disabled, it needs to be rewritten after all the changes here. I'm going to look into rewriting it in a way that sends the results to a visual debugger or something